rust/run_tests.sh

```diff
@@ -0,0 +1,50 @@
+# test common
```

Comment:
I don't think this file should exist? Also, you should add resnet `cargo build` to CI but _not_  `cargo run` unless you want to add an `assert` and update the Dockerfile.

Reply:
I can add the resnet build to CI but this local test is pretty helpful in dev.

Reply:
can't a developer just run the `ci_rust` shell script?

Reply:
Sure, so I'll add the example build for now and later support `cargo run`.

Reply:
Well, since I'm using MXNet resnet the build fails in CI as there's no MXNet CI! as a work around what about pip install it in CI test temporarily?